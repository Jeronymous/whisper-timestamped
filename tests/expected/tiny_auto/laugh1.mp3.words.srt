1
00:00:00,060 --> 00:00:00,600
You

2
00:00:00,600 --> 00:00:00,780
can't

3
00:00:00,780 --> 00:00:00,960
do

4
00:00:00,960 --> 00:00:01,360
it,

5
00:00:01,360 --> 00:00:01,460
you

6
00:00:01,460 --> 00:00:01,680
can't

7
00:00:01,680 --> 00:00:01,780
do

8
00:00:01,780 --> 00:00:01,880
it.

