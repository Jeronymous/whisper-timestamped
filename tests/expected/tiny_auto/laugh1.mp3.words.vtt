WEBVTT

00:00.060 --> 00:00.600
You

00:00.600 --> 00:00.780
can't

00:00.780 --> 00:00.960
do

00:00.960 --> 00:01.360
it,

00:01.360 --> 00:01.460
you

00:01.460 --> 00:01.680
can't

00:01.680 --> 00:01.780
do

00:01.780 --> 00:01.880
it.

