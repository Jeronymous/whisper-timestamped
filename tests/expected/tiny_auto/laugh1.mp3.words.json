{
  "text": " You can't do it, you can't.",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.0,
      "end": 1.7,
      "text": " You can't do it, you can't.",
      "tokens": [
        50364,
        509,
        393,
        380,
        360,
        309,
        11,
        291,
        393,
        380,
        13,
        50464
      ],
      "temperature": 0.0,
      "avg_logprob": -1.3003114553598256,
      "compression_ratio": 0.9310344827586207,
      "no_speech_prob": 0.4180056154727936,
      "confidence": 0.277,
      "words": [
        {
          "text": "You",
          "start": 0.0,
          "end": 0.56,
          "confidence": 0.081
        },
        {
          "text": "can't",
          "start": 0.56,
          "end": 0.82,
          "confidence": 0.302
        },
        {
          "text": "do",
          "start": 0.82,
          "end": 1.04,
          "confidence": 0.118
        },
        {
          "text": "it,",
          "start": 1.04,
          "end": 1.2,
          "confidence": 0.374
        },
        {
          "text": "you",
          "start": 1.26,
          "end": 1.5,
          "confidence": 0.141
        },
        {
          "text": "can't.",
          "start": 1.5,
          "end": 1.7,
          "confidence": 0.868
        }
      ]
    }
  ],
  "language": "en",
  "language_probs": {
    "en": 0.2290046215057373,
    "zh": 0.008879472501575947,
    "de": 0.02778141386806965,
    "es": 0.03680439293384552,
    "ru": 0.018077710643410683,
    "ko": 0.06068018823862076,
    "fr": 0.05030563846230507,
    "ja": 0.027565214782953262,
    "pt": 0.053550075739622116,
    "tr": 0.052719857543706894,
    "pl": 0.02609822154045105,
    "ca": 0.0011647333158180118,
    "nl": 0.012231990694999695,
    "ar": 0.04652511328458786,
    "sv": 0.05790143832564354,
    "it": 0.018219497054815292,
    "id": 0.006198871415108442,
    "hi": 0.02866329252719879,
    "fi": 0.007714619394391775,
    "vi": 0.0017484717536717653,
    "he": 0.006247490178793669,
    "uk": 0.00416172482073307,
    "el": 0.023394295945763588,
    "ms": 0.0023163489531725645,
    "cs": 0.0035045240074396133,
    "ro": 0.003819013712927699,
    "da": 0.004903710447251797,
    "hu": 0.019854480400681496,
    "ta": 0.004097203724086285,
    "no": 0.005219973158091307,
    "th": 0.006808128207921982,
    "ur": 0.03651798143982887,
    "hr": 0.000954348302911967,
    "bg": 0.0005249864771030843,
    "lt": 0.0007880979683250189,
    "la": 0.005219973158091307,
    "mi": 0.002624767366796732,
    "ml": 0.002624767366796732,
    "cy": 0.01487031951546669,
    "sk": 0.00025685937725938857,
    "te": 0.0010814183624461293,
    "fa": 0.006496361456811428,
    "lv": 0.00022229323803912848,
    "bn": 0.0018612387357279658,
    "sr": 0.0006533561972901225,
    "az": 0.0015190986450761557,
    "sl": 0.0012302036629989743,
    "kn": 0.00015218414773698896,
    "et": 0.0005480361869558692,
    "mk": 0.0001639087568037212,
    "br": 0.001776006305590272,
    "eu": 0.001264306134544313,
    "is": 0.0007374632405117154,
    "hy": 0.00030862365383654833,
    "ne": 0.00043864260078407824,
    "mn": 0.0007520083454437554,
    "bs": 0.0013617111835628748,
    "kk": 8.570168574806303e-05,
    "sq": 0.0002609043149277568,
    "sw": 0.001304439501836896,
    "gl": 0.004827686119824648,
    "mr": 0.00020801107166334987,
    "pa": 0.0009395524975843728,
    "si": 0.0011830752482637763,
    "km": 0.0007260299171321094,
    "sn": 0.001996817532926798,
    "yo": 0.0009035589173436165,
    "so": 4.6595199819421396e-05,
    "af": 0.0001509998255642131,
    "oc": 0.00048175465781241655,
    "ka": 0.00022316329705063254,
    "be": 0.000393196678487584,
    "tg": 3.2651128094585147e-06,
    "sd": 0.002860309323295951,
    "gu": 0.00046876014675945044,
    "am": 8.241849718615413e-05,
    "yi": 0.0008259196765720844,
    "lo": 7.160643872339278e-05,
    "uz": 4.3158429718914704e-08,
    "fo": 0.0010941657237708569,
    "ht": 0.001555123133584857,
    "ps": 0.0002712975547183305,
    "tk": 3.5393944131101307e-07,
    "nn": 0.007959508337080479,
    "mt": 0.0004912564181722701,
    "sa": 0.0012892422964796424,
    "lb": 7.492898248528945e-07,
    "my": 0.00042348948772996664,
    "bo": 0.0012398521648719907,
    "tl": 0.011401468887925148,
    "mg": 4.178579331437504e-07,
    "as": 0.0001984855334740132,
    "tt": 1.6100466382340528e-06,
    "haw": 0.004535190761089325,
    "ln": 8.082438580458984e-05,
    "ha": 7.570124580524862e-07,
    "ba": 2.531344307499239e-07,
    "jw": 0.010300369001924992,
    "su": 1.0605447187117534e-06
  }
}